add link to page on getting support if you're disabled (#1289)
DFE-Digital/get-into-teaching-app: app/views/content/funding-your-training.md

  jump_links:
    Tuition fee and maintenance loans: "#tuition-fee-and-maintenance-loans"
    Bursaries and scholarships: "#bursaries-and-scholarships"

>     Extra financial support for parents and carers: "#if-youre-a-parent-or-carer"

Reviewer: @EmmaFrith are you happy with these headings on the right hand side? Given we've amended these below.

Author: oh crap.

* [Adult Dependants’ Grant](https://www.gov.uk/adult-dependants-grant)

## If you're disabled

> You may be able to [get support if you’re a student and you’re disabled, have a learning difficulty or health problem](https://www.gov.uk/disabled-students-allowance-dsa/how-to-claim).

Reviewer 2: ```suggestion
You may be able to [get support if you're a student with a disability, learning difficulty or health problems](https://www.gov.uk/disabled-students-allowance-dsa/how-to-claim).
```

Reviewer: Either feels okay to me - your call @EmmaFrith

Author: I've been using 'if you're disabled' elsewhere on the site in line with the equality and human rights commission style

Reviewer 2: Ah my suggestion was to make the three elements of the list 'equal' rather than rewording them.

It's currently `(disabled and (learning difficulty or health problem)` but I'm guessing it should be `disabled or learning difficulty or health problem`

Reviewer 2: Actually, scratch that. Original makes sense.
